includes/class-newspack-popups.php

```diff
@@ -73,13 +73,13 @@ public static function instance() {
 	 * Constructor.
 	 */
 	public function __construct() {
-
 		// Segmentation requires the main Newspack plugin.
 		self::$segmentation_enabled = class_exists( '\Newspack\Reader_Data' );
 
 		add_action( 'cli_init', [ __CLASS__, 'register_cli_commands' ] );
 
 		add_action( 'init', [ __CLASS__, 'register_cpt' ] );
+		add_action( 'admin_init', [ __CLASS__, 'add_caps' ] );
```

Comment:
This only needs to run once, the roles capabilities are stored in the DB. We don't want to run this on every execution

Reply:
Fixed in 39ba1b9765909ced595a0c675adbcb602f00eaa3. The recommended way to set up caps on plugin activation, but the plugin is already active on sites.